torch/ao/nn/sparse/quantized/linear.py

```diff
@@ -157,26 +159,25 @@ def set_weight_bias(self, w: torch.Tensor, b: Optional[torch.Tensor],
         self._packed_params.set_weight_bias(w, b, row_block_size, col_block_size)
 
     @classmethod
-    def from_float(cls, mod):
+    def from_float(cls, mod, row_block_size=1, col_block_size=4):
         r"""Create a quantized sparse module from a float module.
 
         We only care about the convert at this stage, no need for observers just yet.
+
+        TODO: Need to figure out how to store the block shapes in the mod
```

Comment:
We could have an attr in the sparse quantized linear module to store the sparsity_config